app/javascript/angular/code/services/_sessions_utils.js

```diff
@@ -97,6 +97,15 @@ export const sessionsUtils = (
     createSessionData(session, data);
     session.loaded = true;
     callback();
+  },
+  refreshMapView: function(sessions) {
```

Comment:
how about using an arrow function?

Reply:
That's a good point. I guess I'll change the whole file then.

Reply:
Rysiek mentioned doing that later too, so nice that you wanna take care of it right away:)

Reply:
Turns out it's not that easy and breaks some test, so I'm gonna stick with non-arrow functions 🙁